queries/models/monitoramento/schema.yml

```
@@ -700,3 +700,76 @@
+      - name: indicadores
+        description: "Indicadores de regularidade do sistema de ar condicionado e temperatura"
+      - name: quantidade_dia_falha_operacional
+        description: "Quantidade consecutiva de dias com falha operacional do ar condicionado"
+      - name: motivo
+        description: "Descrição do motivo da falha do sistema de ar condicionado"
+      - name: datetime_ultima_atualizacao
+        description: "{{ doc('datetime_ultima_atualizacao') }}"
+      - name: versao
+        description: "{{ doc('versao') }}"
+      - name: id_execucao_dbt
+        description: "{{ doc('id_execucao_dbt') }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)